modules/pel/peltool/peltool.py

```diff
@@ -245,6 +245,78 @@ def parseAndWriteOutput(file: str, output_dir: str, config: Config,
             print(f"No PEL parsed for {file}: {e}")
 
 
+def parsePELSummary(stream: DataStream, config: Config):
+    """
+    Parses and summarizes data from a PEL stream.
+    Returns: Event ID (eid) and summary of PEL data.
+            If no PEL is parsed or an error occurs, empty strings are returned.
+    """
+    out = OrderedDict()
+    ret, ph = generatePH(stream, out)
+    if ret is False:
+        return "", ""
+    eid = ph.pLID
+    ret, uh = generateUH(stream, ph.creatorID, out)
+    if ret is False:
+        return "", ""
+    if not uh.isServiceable():
+        return "", ""
+    
+    summary = OrderedDict()
+    for _ in range(2, ph.sectionCount):
+        sectionID, sectionLen, versionID, subType, componentID = parseHeader(stream)
+        section_json = OrderedDict()
+        sectionFun(stream, section_json, sectionID, sectionLen,
+                   versionID, subType, componentID, ph.creatorID, config)
+        if sectionID == SectionID.primarySRC.value or sectionID == SectionID.secondarySRC.value:
```

Comment:
you can remove the SecondarySRC check here.  in this case, we'll just want to show the summary for the primary SRC section